Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.h

```diff
@@ -43,32 +46,52 @@ namespace ROS2::Utils
         AZ::Uuid m_sourceGuid = AZ::Uuid::CreateNull();
     };
 
+    //! Status of the copy process.
+    enum CopyStatus
+    {
+        Unresolvable, //! Unresolvable
+        Waiting, //! Waiting for copy
+        Copying, //! Copying
+        Copied, //! Copied
+        Exists, //! Already exists
+        Failed, //! Failed
+    };
+
+    //! Bitfield containing the types of asset references are associated with a given unresolved URI or path reference.
+    //! These are flags because the same mesh URI can refer to both a Visual and a Collider entry, for example.
+    enum class ReferencedAssetType
```

Comment:
This code is already in code base, why it is marked as part of this contribution.